backend/shared/src/update-contract-views.ts

```diff
@@ -7,7 +7,7 @@ import { JobContext } from 'shared/utils'
 import { uniq } from 'lodash'
 import { SafeBulkWriter } from 'shared/safe-bulk-writer'
 
-export async function updateContractViews({ log, lastEndTime }: JobContext) {
+export async function updateContractViews({ log }: JobContext) {
```

Comment:
I don't think caching this `views` data will be important at all anymore using the new faster table, so in a future PR I will probably remove this job and query it on demand.